src/ImageSharp/Formats/Jpeg/Components/Block8x8F.cs

```diff
@@ -543,15 +543,14 @@ public override string ToString()
         {
             var sb = new StringBuilder();
             sb.Append('[');
-            for (int i = 0; i < Size; i++)
+            for (int i = 0; i < Size - 1; i++)
```

Comment:
Maybe you could also 'calculate' `Size-1` once?

Reply:
I could, but this is done by the compiler: Size is constant, so Size-1 is constant, and in fact the compiler simplifies the term.
Thus it's not only calculated once, it's calculated at compile time and the intermediate variable, used as VARIABLE would even harm performance instead.

Reply:
minimized example on what the compiler does (even in debug mode, but release is sligthly more concise): https://sharplab.io/#v2:EYLgHgbALANALiAlgGwD4AEDMACdBGCXAJmwGEBYAKAG8rt7sBjAewDsBnObRVrgZUQAvAKbYAvNmgBuOgyy4CuKNgCyAQx4AKAA5qATmoC27BQAYA2gF1s+gObsAlLPq1KDd9gBmzPZp5dEMVMpbgAeAREAWjwQxABqOIdsVw8GAF9nbAzKNKA=

you see the constant value 63 in IL_0009 being loaded as ldc.i4.s 63

Reply:
Thanks for the explanation.